sdk/management/docs/AUTH.md

```diff
@@ -0,0 +1,92 @@
+
+## Prerequisites
+
+Before authenticating the Azure client, there are a few authentication classes you need to know before you decide which kind of authentication would meet your requirement.
+
+* TokenCredential
+  * The `TokenCredential` is an interface in the `azure-core` package for credentials that can provide a token. 
+  * Azure Identity offers multiple implementations of the `TokenCredential` class in the `azure-identity` package. To learn more, see [credentials in Azure Identity](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/identity/azure-identity#credentials).
+
+Sample code to create a simple `ClientSecretCredential`:
+
+```java
+ClientSecretCredential clientSecretCredential = new ClientSecretCredentialBuilder()
+    .clientId("<YOUR_CLIENT_ID>")
+    .clientSecret("<YOUR_CLIENT_SECRET>")
+    .tenantId("<YOUR_TENANT_ID>")
+    .build();
+```
+
+* AzureProfile
+  * The `AzureProfile` is a class holding `AzureEnvironment`, `subscriptionId`, `tenantId` to configure the requests sending to wire. 
+  * The `subscriptionId` is mandantory for most resource management while the `tenantId` would be required only for Graph RBAC. They can be set via [environment variables](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/identity/azure-identity#environment-variables).
+
+Sample code to create a `AzureProfile`:
+
+```java
+//AzureProfile profile = new AzureProfile(AzureEnvironment.AZURE, true);
+AzureProfile profile = new AzureProfile("<YOUR_TENANT_ID>", "<YOUR_SUBSCRIPTION_ID>", AzureEnvironment.AZURE);
+```
+
+* HttpPipelinePolicy
+  * The `HttpPipelinePolicy` is an interface that process provided request context and invokes the next policy. To learn more, see [policy in Azure Core](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/core/azure-core/src/main/java/com/azure/core/http/policy) and [policy in the library](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/resources/mgmt/src/main/java/com/azure/management/resources/fluentcore/policy).
+
+
+* HttpClient
+  * The `HttpClient` is a generic interface for sending HTTP requests and getting responses. 
+  * [azure-core-http-netty](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/core/azure-core-http-netty) provides a Netty derived HTTP client.
+  * [azure-core-http-okhttp](https://github.com/Azure/azure-sdk-for-java/tree/master/sdk/core/azure-core-http-okhttp) provides an OkHttp derived HTTP client.
+
+
+* HttpPipeline
+  * The `HttpPipeline` is a class that HTTP requests and responses will flow through. It is a construct that contains a list of `HttpPipelinePolicy` which are applied to a request sequentially to prepare it being sent by an `HttpClient`.
+
+Sample code to create a `HttpPipeline`:
+
+```java
+HttpPipeline httpPipeline = new HttpPipelineBuilder()
+    .policies(httpPipelinePolicies)
+	.httpClient(httpClient)
+	.build();
+```
+
+## Basic Authentication
```

Comment:
Maybe "Simple Authentication"? *Basic* could be confused with "Basic AUTH" https://tools.ietf.org/html/rfc7617